felix/fv/ipip_test.go

```diff
@@ -129,15 +128,6 @@ var _ = infrastructure.DatastoreDescribe("_BPF-SAFE_ IPIP topology before adding
 		infra.Stop()
 	})
 
-	It("should use the --random-fully flag in the MASQUERADE rules", func() {
```

Comment:
Is this intentional? Do we need to remove this test as part of the change?

Reply:
I'm sorry, it's a typo here, Now this has been fixed.